packages/libs/client/src/signing/eckoWallet/eckoTypes.ts

```diff
@@ -0,0 +1,73 @@
+import { ICommand } from '@kadena/types';
+
+import { IQuicksignResponseOutcomes } from '../../signing-api/v1/quicksign';
+import { ISignFunction, ISingleSignFunction } from '../ISignFunction';
+
+import { connect, isConnected, isInstalled } from './eckoCommon';
+
+export type EckoStatus = 'success' | 'fail';
+
+/**
+ * Interface that describes the common functions to be used with Ecko Wallet
+ * @public
+ */
+export interface ICommonEckoFunctions {
```

Comment:
its better to keep the type file independent form the implementations so I would write this interface like this 
```
export interface ICommonEckoFunctions {
  isInstalled: () => boolean
  isConnected: (networkId:string) => Promise<boolean>
  connect: (networkId:string) => Promise<boolean>;
}

Reply:
I modified the interface and changed `eckoCommon` to use the types of the functions from the interface